CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing
+
+When contributing to this repository, please if possible first discuss the change you wish to make via issue,
+email, slack or any other method with the owners of this repository before making a change.
+If it's not possible then please provide descriptive explanantion together with your Pull Request.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Update all manifests that may be affected by change provided in Pull Request
+4. Run bazel build process (`bazel run //:gazelle`) and commit generated changes.
```

Comment:
IMHO that should be automatically checked if this is something that is required and described in DEVELOPMENT.md or README.md